cmd/bootstrap/main.go

```diff
@@ -43,6 +44,7 @@ type config struct {
 	ESConsumerName string `env:"MF_BOOTSTRAP_EVENT_CONSUMER"   envDefault:"bootstrap"`
 	ThingsURL      string `env:"MF_THINGS_URL"                 envDefault:"http://localhost:9000"`
 	JaegerURL      string `env:"MF_JAEGER_URL"                 envDefault:"localhost:6831"`
+	InstanceID     string `env:"MF_BOOTSTRAP_INSTANCE_ID"           envDefault:""`
 }
```

Comment:
`envDefault` of Instance ID is not algined with others 

```suggestion
	InstanceID     string `env:"MF_BOOTSTRAP_INSTANCE_ID"      envDefault:""`
}
```